hack/templates/resources/90-shoot.yaml.tpl

```diff
@@ -254,6 +254,9 @@ spec:
   #     syncPeriod: 30s
   #     tolerance: 0.1
   #     upscaleDelay: 1m0s
+  #     downscaleStabilization: 5m0s
```

Comment:
Can we add a comment here to make it clearer to the user which flags can be used?
For example:

```yaml
  #   horizontalPodAutoscaler:
  #     syncPeriod: 30s
  #     tolerance: 0.1
  ## only usable with Kubernetes < 1.12
  #     downscaleDelay: 15m0s
  #     upscaleDelay: 1m0s
  ## only usable with Kubernetes >= 1.12
  #     downscaleStabilization: 5m0s
  #     initialReadinessDelay: 30s
  #     cpuInitializationPeriod: 5m0s
```

Similarly, could you please update the `.spec.kubernetes.kubeAPIServer.oidcConfig` part, for example:

```yaml
   #   oidcConfig:
   #     caBundle: |
   #       -----BEGIN CERTIFICATE-----
   #       Li4u
   #       -----END CERTIFICATE-----
   #     clientID: client-id
   #     groupsClaim: groups-claim
   #     groupsPrefix: groups-prefix
   #     issuerURL: https://identity.example.com
   #     usernameClaim: username-claim
   #     usernamePrefix: username-prefix
   ## only usable with Kubernetes >= 1.10
   #     signingAlgs: RS256,some-other-algorithm
   ## only usable with Kubernetes >= 1.11
   #     requiredClaims:
   #       key: value
```